android/capacitor/build.gradle

```diff
@@ -41,6 +42,8 @@ repositories {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support:support-compat:27.+'
```

Comment:
I prefer using `27.1.1` as it's the version used in the other support libraries we have.
Using `27+` will require less updates, but can also be less predictable.

In case we stick to `27+` then we should use that for all the libraries we have.